java/src/main/java/com/logicalclocks/hsfs/engine/HudiEngine.java

```diff
@@ -168,7 +170,7 @@ private FeatureGroupCommit getLastCommitMetadata(SparkSession sparkSession, Stri
     }
 
     // table name
-    String tableName = utils.getTableName(featureGroup);
+    String tableName = featureGroup.getName() + "_" + featureGroup.getVersion();
```

Comment:
Use this method instead: https://github.com/logicalclocks/feature-store-api/blob/7c704e06ac1ede40a01895d6b502f449fd028861/java/src/main/java/com/logicalclocks/hsfs/engine/Utils.java#L112